sigal/gallery.py

```diff
@@ -152,6 +152,7 @@ def _get_metadata(self):
         """ Get image metadata from filename.md: title, description, meta."""
         self.description = ''
         self.meta = {}
+        self.file_metadata = {}
```

Comment:
Does it need to be a dict ? It seems that it could be an attribute of the Media object.

Reply:
My design here is far from perfect, indeed.

I started with a simple dict, because it is flexible enough. But we could imagine a new class `MediaMetadata` that hold some functions currently in `image.py`.
But I still don't know if we need one metadata holder for each underlying file (`src`, `dst` and `thumb`), or one for all.